src/main/java/mate/academy/spring/service/mapper/MovieSessionDtoMapper.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.model.dto.request.MovieSessionRequestDto;
+import mate.academy.spring.model.dto.response.MovieSessionResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MovieSessionDtoMapper {
+    public MovieSessionResponseDto toDto(MovieSession movieSession) {
+        MovieSessionResponseDto responseDto = new MovieSessionResponseDto();
+        responseDto.setId(movieSession.getId());
+        responseDto.setMovie(movieSession.getMovie());
```

Comment:
the best practices is 
```suggestion
        responseDto.setMovie(movieSession.getMovie().getId());
```